source/change-streams/tests/unified/change-streams.yml

```diff
@@ -140,7 +140,50 @@ tests:
   - description: "Test that comment is set on getMore"
     runOnRequirements:
       - minServerVersion: "4.4.0"
-        topologies: [ single, replicaset ]
```

Comment:
I've noted that `single` was introduced by https://github.com/mongodb/specifications/commit/ad1ee9b764f7d9ff193fe485d31feca536e6a2d7; however, this had no effect as the top-level `runOnRequirements` specified replica sets and sharded clusters only (https://github.com/mongodb/specifications/commit/962fb5f20e9e31b528319fe245147928a3100411#diff-ba3c94557f77e83f167fb27ed81eb223e4eb3e6178238f3d93def75f37adb301R3).